plugins/org.yakindu.base.gmf.runtime/src/org/yakindu/base/gmf/runtime/highlighting/HighlightingSupportAdapter.java

```diff
@@ -195,12 +190,7 @@ public synchronized void releaseEditor() {
 			throw new IllegalStateException("Editor not locked!");
 		}
 		List<Action> singletonList = new ArrayList<>();
-		singletonList.add(new Action() {
-			@Override
-			public void execute(IHighlightingSupport hs) {
-				releaseInternal();
-			}
-		});
+		singletonList.add((support) -> releaseInternal());
```

Comment:
Name method 'releaseEditorInternal' instead of 'releaseInternal' for consistency with 'lockEditorInternal'?

Reply:
Done.